lib/workers/repository/process/lookup/index.ts

```diff
@@ -327,21 +323,20 @@ export async function lookupUpdates(
       logger.debug(
         `Dependency ${packageName} has unsupported/unversioned value ${currentValue} (versioning=${config.versioning})`
       );
+
       if (!pinDigests && !currentDigest) {
         res.skipReason = 'invalid-value';
       } else {
         delete res.skipReason;
       }
-    } else if (!currentValue && isReplacementNameRulesConfigured(config)) {
-      logger.debug(
-        `Handle name-only replacement for ${packageName} without current version`
-      );
-
-      addReplacementUpdateIfValid(res.updates, config);
     } else {
       res.skipReason = 'invalid-value';
     }
 
+    if (isReplacementRulesConfigured(config)) {
```

Comment:
this now has maybe the skip reason set from above, so it will probably fully ignored later

Reply:
hmmm, good callout.  perhaps https://github.com/renovatebot/renovate/pull/21151 is "safer"

Reply:
actually, i think this makes it "ok" 

https://github.com/renovatebot/renovate/blob/7f9874c8998e10c523fff1234c66ae4e6c327ab3/lib/workers/repository/process/lookup/index.ts#L424-L426